src/OBD2.cpp

```diff
@@ -695,6 +695,36 @@ int OBD2Class::supportedPidsRead()
   return 1;
 }
 
+int OBD2Class::clearAllStoredDTC() {
```

Comment:
We should add an entry for `clearAllStoredDTC` in both `keywords.txt` and `API.md`.

Reply:
I agree.  I'll add the function to the keywords.txt file.  How do you feel about this for the API.md?

### Clear Stored Diagnostic Trouble Codes (DTC)

Clears any stored DTCs in the vehicle ECU, and turns off Malfunction Indicator Light (MIL, more commonly known as the Check Engine light).

```arduino
int success = OBD2.clearAllStoredDTC();
```

Returns 1 if successful, or 0 on failure